mappings/net/minecraft/world/gen/blockpredicate/MultiBlockPredicate.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_6648 net/minecraft/world/gen/blockpredicate/MultiBlockPredicate
```

Comment:
Hmm, I know it holds multiple blocks, but it sounds like a predicate for multiblocks